apps/script/checkEntryPoints.js

```diff
@@ -84,7 +78,7 @@ const SILENCED = [
   'netsim',
   'pd',
   'plc',
-  'publicKeyCryptography',
+  'projectsList',
```

Comment:
Admittedly I only tried to work around this by looking at how a couple other examples dealt with this, and they all dealt with it by having an entry in this list. I will try harder to avoid adding to this list. If you know of an example that avoids this, please let me know.